ch_6_GPU_Acceleration/6_3_GPU_Whisper-medium.md

```diff
@@ -0,0 +1,131 @@
+
+
+## 6.2.2 Load Model in Low Precision
+
+One common use case is to load a model from Hugging Face with IPEX-LLM low-bit precision optimization. For Whisper (medium), you could simply import `ipex_llm.transformers.AutoModelForSpeechSeq2Seq` instead of `transformers.AutoModelForSpeechSeq2Seq`, and specify `load_in_4bit=True` parameter accordingly in the `from_pretrained` function.
+
+For Intel GPUs, **once you have the model in low precision, set it to `to('xpu')`.**
+
+**For INT4 Optimizations (with `load_in_4bit=True`):**
+
+```python
+from ipex_llm.transformers import AutoModelForSpeechSeq2Seq
+
+model_in_4bit = AutoModelForSpeechSeq2Seq.from_pretrained(pretrained_model_name_or_path="openai/whisper-medium",
+                                                  load_in_4bit=True)
+model_in_4bit_gpu = model_in_4bit.to('xpu')
+```
+
+> [!NOTE]
+> * IPEX-LLM has supported `AutoModel`, `AutoModelForCausalLM`, `AutoModelForSpeechSeq2Seq` and `AutoModelForSeq2SeqLM`, etc.
+>
+>   If you have already downloaded the Whisper (medium) model, you could specify `pretrained_model_name_or_path` to the model path.
+>
+> * Currently, `load_in_low_bit` supports options `'sym_int4'`, `'asym_int4'`, `'sym_int8'`, `'nf4'`, `'fp6'`, `'fp8'`,`'fp16'`, etc., in which `'sym_int4'` means symmetric int 4, `'asym_int4'` means asymmetric int 4, and `'nf4'` means 4-bit NormalFloat, etc. Relevant low bit optimizations will be applied to the model.
+>
+>   `load_in_4bit=True` is equivalent to `load_in_low_bit='sym_int4'`.
+>
+> * When running LLMs on Intel iGPUs for Windows users, we recommend setting `cpu_embedding=True` in the from_pretrained function.
+> 
+>   This will allow the memory-intensive embedding layer to utilize the CPU instead of iGPU.
+>
+> * You could refer to the [API documentation](https://ipex-llm.readthedocs.io/en/latest/doc/PythonAPI/LLM/transformers.html) for more information.
+
+## 6.2.3 Load Whisper Processor
+
+A Whisper processor is also needed for both audio pre-processing, and post-processing model outputs from tokens to texts. IPEX-LLM does not provide a customized implementation for that, so you might want to use the official `transformers` API to load `WhisperProcessor`:
+
+```python
+from transformers import WhisperProcessor
+
+processor = WhisperProcessor.from_pretrained(pretrained_model_name_or_path="openai/whisper-medium")
+```
+
+> [!NOTE]
+> If you have already downloaded the Whisper (medium) model, you could specify `pretrained_model_name_or_path` to the model path.
+
+## 6.2.4 Run Model to Transcribe English Audio
+
+Once you have optimized the Whisper model using IPEX-LLM with INT4 optimization and loaded the Whisper processor, you are ready to begin transcribing the audio through model inference.
+
+Let's start with the English audio file `audio_en.wav`. Before we feed it into Whisper processor, we need to extract sequence data from raw speech waveform:
```

Comment:
Please add some note here about the source of `audio_en.wav` (such as, from dataset voxpopuli?), and "audio_zh.wav" also needs.